kwave/utils/io.py

```diff
@@ -46,20 +46,20 @@ def get_h5_literals():
         'HDF_FILE_MAJOR_VERSION': '1',
         'HDF_FILE_MINOR_VERSION': '2',
 
-        # compression level
-        'HDF_COMPRESSION_LEVEL': 0
+        # compression level: set to be same as default h5py
+        'HDF_COMPRESSION_OPTIONS': 4
     })
     return literals
 
 
-def write_matrix(filename, matrix: np.ndarray, matrix_name: str, compression_level:int =None, auto_chunk: bool =True):
+def write_matrix(filename, matrix: np.ndarray, matrix_name: str, compression_options: Union[int, str] = None, auto_chunk: bool = True):
     # get literals
     h5_literals = get_h5_literals()
 
     assert isinstance(auto_chunk, bool), "auto_chunk must be a boolean."
 
-    if compression_level is None:
-        compression_level = h5_literals.HDF_COMPRESSION_LEVEL
+    if compression_options is None:
```

Comment:
Instead of None being a possible value, why not set a default value in the signature?